DERP. Linter doesn't  like these comments 😂 

```
libnetwork/libnetwork_linux_test.go:306:4: Duplicate words (network) found (dupword)
			// 	network driver options are not supported if the network depends on a configuration network
			^
libnetwork/libnetwork_linux_test.go:825:2: Duplicate words (map[]) found (dupword)
	//	invalid Sandbox passed to endpoint leave: &{  {{    []} {   [] [] []} map[] false false []} [] <nil> <nil> <nil> {{{} 0} {0 0}} [] map[] map[] <nil> 0 false false false false false []  {0 0} {0 0}}
	^
libnetwork/libnetwork_linux_test.go:1126:2: Duplicate words (map[]) found (dupword)
	//	invalid Sandbox passed to endpoint join: &{  {{    []} {   [] [] []} map[] false false []} [] <nil> <nil> <nil> {{{} 0} {0 0}} [] map[] map[] <nil> 0 false false false false false []  {0 0} {0 0}}
	^
libnetwork/libnetwork_linux_test.go:555:2: ineffectual assignment to g (ineffassign)
	g, err := controller.NetworkByID("network1")
	^
```

--- a/libnetwork/libnetwork_linux_test.go
+++ b/libnetwork/libnetwork_linux_test.go
@@ -132,7 +132,10 @@ func TestNilRemoteDriver(t *testing.T) {
 
 	_, err := controller.NewNetwork("framerelay", "dummy", "",
 		libnetwork.NetworkOptionGeneric(getEmptyGenericOption()))
+
+	// TODO(thaJeztah): should attempting to use a non-existing plugin/driver return an [errdefs.InvalidParameter] ?
 	assert.Check(t, is.ErrorType(err, errdefs.IsNotFound))
+	assert.Check(t, is.Error(err, "could not find plugin framerelay in v1 plugin registry: plugin not found"))
 }
 
 func TestNetworkName(t *testing.T) {

--- a/libnetwork/libnetwork_linux_test.go
+++ b/libnetwork/libnetwork_linux_test.go
@@ -109,12 +109,10 @@ func TestNull(t *testing.T) {
 
 	// host type is special network. Cannot be removed.
 	err = network.Delete()
-	if err == nil {
-		t.Fatal(err)
-	}
-	if _, ok := err.(types.ForbiddenError); !ok {
-		t.Fatalf("Unexpected error type")
-	}
+
+	// TODO(thaJeztah): should this be an [errdefs.ErrInvalidParameter] ?
+	assert.Check(t, is.ErrorType(err, errdefs.IsForbidden))
+	assert.Check(t, is.Error(err, `network of type "null" cannot be deleted`))
 }
 
 func TestUnknownDriver(t *testing.T) {

--- a/libnetwork/libnetwork_linux_test.go
+++ b/libnetwork/libnetwork_linux_test.go
@@ -150,13 +150,7 @@ func TestNetworkName(t *testing.T) {
 	}
 
 	_, err := createTestNetwork(controller, bridgeNetType, "", netOption, nil, nil)
-	if err == nil {
-		t.Fatal("Expected to fail. But instead succeeded")
-	}
-
-	if _, ok := err.(libnetwork.ErrInvalidName); !ok {
-		t.Fatalf("Expected to fail with ErrInvalidName error. Got %v", err)
-	}
+	assert.Check(t, is.ErrorType(err, errdefs.IsInvalidParameter), "Expected to fail with ErrInvalidName error")
 
 	const networkName = "testnetwork"
 	n, err := createTestNetwork(controller, bridgeNetType, networkName, netOption, nil, nil)

